packages/react-components/react-combobox/docs/Migration.md

```diff
@@ -0,0 +1,176 @@
+# @fluentui/react-combobox Migration Guide
+
+## Migration from v8 Dropdown
+
+The v8 Dropdown component should be replaced with the [v9 Select](https://github.com/microsoft/fluentui/blob/master/packages/react-select/Spec.md) or v9 Dropdown component. The biggest change is that the v8 `options: IDropdownOption[]` prop should be mapped to children in v9.
+
+### Property mapping
+
+| v8 Dropdown            | v9 Dropdown               |
+| ---------------------- | ------------------------- |
+| `options`              | `<Option>` children       |
+| `ariaLabel`            | Native `aria-*` props     |
+| `calloutProps`         | `positioning` + `listbox` |
+| `className`            | `className`               |
+| `componentRef`         | `ref`                     |
+| `defaultSelectedKeys`  | defaultSelectedOptions    |
+| `disabled`             | `disabled`                |
+| `dropdownWidth`        | `listbox` slot styles     |
+| `errorMessage`         | Handled by Field          |
+| `id`                   | `id`                      |
+| `label`                | Handled by Field          |
+| `multiSelect`          | `multiselect`             |
+| `multiSelectDelimiter` | controlled `value`        |
+| `notifyOnReselect`     | `onClick` on Option       |
+| `onChange`             | `onOptionSelect`          |
+| `onDismiss`            | `onOpenChange`            |
+| `onRenderCaretDown`    | `expandIcon` slot         |
+| `onRenderContainer`    | `listbox` slot            |
+| `onRenderLabel`        | Handled by Field          |
+| `onRenderItem`         | children                  |
+| `onRenderList`         | `listbox` slot            |
+| `onRenderOption`       | `<Option>` children       |
+| `onRenderPlaceholder`  | `button` slot             |
+| `onRenderTitle`        | `button` slot             |
+| `openOnKeyboardFocus`  | controlled `open`         |
+| `panelProps`           | N/A                       |
+| `placeholder`          | `placeholder`             |
+| `required`             | `required`                |
+| `selectedKey(s)`       | `selectedOptions`         |
+| `styles`               | `styles`                  |
```

Comment:
```suggestion
```

Isn't the `styles` prop a v8-specific styling hook that is removed in v9?

Reply:
It's not a hook, but an object that needs to match a component-specific interface for overriding styles: https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox#IComboBoxStyles

I included it in the prop table b/c it's part of the v8 interface, but I'll update the v9 mapping to be "`className` or HTML `style` prop".

Reply:
actually ended up with `makeStyles` or the HTML `style` prop`